### ISSUE TITLE
Missing icon error in log

### ISSUE THREAD

**Issue by mhutch** [mono/monodevelop#4973]
The following missing icon error shows up in the log. Looks like fallout from structured build output.

```
ERROR [2018-05-31 18:32:37Z]: Error loading icon 'md-build-combine'
System.InvalidOperationException: Resource 'build-solution-16.png' not found in add-in 'MonoDevelop.Ide,7.6'
   at Mono.Addins.RuntimeAddin.GetResource(String resourceName, Boolean throwIfNotFound) in C:\Users\Mikayla\code\monodevelop\main\external\mono-addins\Mono.Addins\Mono.Addins\RuntimeAddin.cs:line 534
   at MonoDevelop.Ide.CustomImageLoader.LoadImage(String fileName) in C:\Users\Mikayla\code\monodevelop\main\src\core\MonoDevelop.Ide\MonoDevelop.Ide\ImageService.cs:line 901
   at Xwt.Drawing.StreamImageLoader.LoadImage(String fileName) in C:\Users\Mikayla\code\monodevelop\main\external\xwt\Xwt\Xwt.Drawing\Image.cs:line 1099
   at Xwt.Drawing.Image.LoadImage(ImageLoader loader, String fileName, ImageTagSet tagFilter) in C:\Users\Mikayla\code\monodevelop\main\external\xwt\Xwt\Xwt.Drawing\Image.cs:line 193
   at Xwt.Drawing.Image.FromCustomLoader(IImageLoader loader, String fileName, ImageTagSet tags) in C:\Users\Mikayla\code\monodevelop\main\external\xwt\Xwt\Xwt.Drawing\Image.cs:line 379
   at Xwt.Drawing.Image.FromCustomLoader(IImageLoader loader, String fileName) in C:\Users\Mikayla\code\monodevelop\main\external\xwt\Xwt\Xwt.Drawing\Image.cs:line 369
   at MonoDevelop.Ide.ImageService.LoadStockIcon(RuntimeAddin addin, String stockId, String resource, String imageFile, String iconId, IconSize iconSize, String animation, Boolean forceWildcard) in C:\Users\Mikayla\code\monodevelop\main\src\core\MonoDevelop.Ide\MonoDevelop.Ide\ImageService.cs:line 117
```

> VS bug [#625262](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/625262)

**Comment by mhutch** [comment]
@slluis @rodrmoya when we fix GH issues, can we please make sure we link the commit that fixed it?

**Comment by xamarin-release-manager** [comment]
> [sync] [VS-12] Comment by **xamarinc**



Fixed in version 8.6.0.802 (master)



Author: Rodrigo Moya  
 Commit: 8e04f7b8cf0cd8075ac63491abf43c35773d0edd (xamarin/monodevelop)
